integration-tests/chopsticks/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@kiltprotocol/e2e-tests",
+  "version": "0.0.1",
+  "description": "chopsticks integration tests",
+  "private": "true",
+  "type": "module",
+  "repository": "git@github.com:KILTprotocol/kilt-node.git",
+  "author": "[\"KILT <info@kilt.io>\"]",
+  "license": "MIT",
+  "devDependencies": {
+    "@acala-network/chopsticks": "0.10.0",
+    "@acala-network/chopsticks-testing": "0.10.1",
+    "@polkadot/api": "^10.11.2",
+    "@types/node": "^20.11.30",
+    "@typescript-eslint/eslint-plugin": "^7.7.0",
+    "@typescript-eslint/parser": "^7.7.0",
+    "eslint": "^8.0.1",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-config-standard-with-typescript": "^43.0.1",
+    "eslint-plugin-import": "^2.25.2",
+    "eslint-plugin-n": "^15.0.0 || ^16.0.0 ",
+    "eslint-plugin-prettier": "^5.1.3",
+    "eslint-plugin-promise": "^6.0.0",
+    "prettier": "^3.2.5",
+    "ts-node": "^10.9.2",
+    "tsx": "^4.7.1",
+    "typescript": "*",
+    "vitest": "^1.4.0",
+    "eslint-plugin-jsx-a11y": "^6.8.0"
+  },
+  "scripts": {
+    "lint": "eslint src && prettier --check src",
+    "lint:fix": "eslint --fix src && prettier --write src",
+    "clean": "rm -rf ./db && cargo build -p spiritnet-runtime",
+    "test": "LOG_LEVEL=error vitest",
+    "test:CI": "vitest --bail 0  --no-file-parallelism"
```

Comment:
If `--no-file-parallelism` isn't specified, vitest executes all three test files concurrently. This includes spawning three services for each test case. In the logs, I noticed that some services aren't responding, likely due to insufficient computational resources. Enabling this flag ensures that tests run sequentially, preventing individual service failures during initialization.

I retried the tests 3 times while other tests were running in the background. All three succeeded.

Reply:
I think to me this calls for suboptimal test setup then. Did you figure out if it's something we do or something Chopsticks does? I did not find the same flag on their e2e tests, so I suspect it could be some concurrency issue. Anyway, not blocking this merge, but it can perhaps be tackled as part of https://github.com/KILTprotocol/ticket/issues/3345.